README.md

```diff
@@ -21,6 +21,19 @@ To run the project you may specify a `.toml` configuration file, if none is spec
 ```
 A sample configuration file can be found at `config.toml.template`
 
+### Meta Bucket
+
+A bucket to store shared configuration across multiple `MinSQL` instances is needed, the location and access to it should be configured on the `config.toml`.
+
+````toml
+[server]
+   ...
+   metadata_bucket = "meta"
```

Comment:
This names too generic, prefix this with MinSQL? Like `minsql-meta`

Reply:
Also create a bucket automatically if not specified ? Such as default to `minsql-meta` if not specified ?

Reply:
you mean the name of the bucket? It's a configuration so they can choose the name of the bucket on their own, the example is meta.
Also we can definitively make it so it auto creates the meta bucket, if it exists, for that purpose choosing a default meta bucket name makes sense, however if they are running multiple MinSQL clusters, maybe we want them to think a little bit about the meta bucket name.

Reply:
Even when we document we should say `minsql-meta` most users will choose `meta` which is too generic, or specify how they should choose a bucket based on multiple MinSQL clusters.

Reply:
that makes sense

Reply:
I did the change btw